packages/@aws-cdk/aws-ecs/lib/linux-parameters.ts

```diff
@@ -19,6 +19,32 @@ export interface LinuxParametersProps {
    * @default No shared memory.
    */
   readonly sharedMemorySize?: number;
+
+  /**
+   * The total amount of swap memory (in MiB) a container can use. This parameter
+   * will be translated to the --memory-swap option to docker run.
+   *
+   * This parameter is only supported when you are using the EC2 launch type.
+   * Accepted values are positive integers.
+   *
+   * @default No swap.
+   */
+  readonly maxSwap?: number;
```

Comment:
This type should be `core.Size`. Sorry I didn't think of this in the first round of reviews!